server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheContainerCountersResource.java

```diff
@@ -0,0 +1,73 @@
+package org.jboss.as.clustering.infinispan.subsystem;
+
+import org.jboss.as.controller.AttributeDefinition;
+import org.jboss.as.controller.PathElement;
+import org.jboss.as.controller.ReloadRequiredRemoveStepHandler;
+import org.jboss.as.controller.SimpleAttributeDefinition;
+import org.jboss.as.controller.SimpleAttributeDefinitionBuilder;
+import org.jboss.as.controller.SimpleResourceDefinition;
+import org.jboss.as.controller.registry.AttributeAccess;
+import org.jboss.as.controller.registry.ManagementResourceRegistration;
+import org.jboss.as.controller.services.path.ResolvePathHandler;
+import org.jboss.dmr.ModelType;
+
+/**
+ * CacheContainerCountersResource
+ *
+ * @author Vladimir Blagojevic
+ * @since 9.2
+ */
+public class CacheContainerCountersResource extends SimpleResourceDefinition {
+    static final PathElement PATH = PathElement.pathElement(ModelKeys.COUNTERS, ModelKeys.COUNTERS_NAME);
+
+    //TODO Could not find these definitions anywhere in counter API
+    private static final String AVAILABLE = "AVAILABLE";
+    private static final String CONSISTENT = "CONSISTENT";
```

Comment:
These constants are defined in the following enum `org.infinispan.counter.configuration.Reliability`